ProjectLighthouse/Pages/SlotPage.cshtml.cs

```diff
@@ -43,6 +45,19 @@ public async Task<IActionResult> OnGet([FromRoute] int id)
             this.Comments = new List<Comment>();
         }
 
+        if (this.ReviewsEnabled)
+        {
+            this.Reviews = await this.Database.Reviews.Include(r => r.Reviewer)
+                .OrderByDescending(r => r.Timestamp)
```

Comment:
```suggestion
                .OrderByDescending(r => r.ThumbsUp - r.ThumbsDown)
                .ThenByDescending(r => r.Timestamp)
```
This should be ordered by either thumbsUp or the result of thumbs up minus thumbs down